app/src/main/java/org/covidwatch/android/data/contactevent/ContactEventFetcher.kt

```diff
@@ -15,6 +15,16 @@ interface ContactEventFetcher {
      * @return The resulting contact events.
      * @exception Exception Throws an exception on error.
      */
-    fun fetch(timeWindow: ClosedRange<Date>,
-              resultCb: List<String>.(infectionState: InfectionState) -> Unit)
+    fun fetch(timeWindow: ClosedRange<Date>)
```

Comment:
need to remove `resultDb` from docs

Reply:
I honestly don't see any reason to use `Range` for dates here. We could perfectly use separate `start`, `end` parameters. And in the `handleQueryResult` to have only `end` time because this what we essently only using there.